docs/install.md

```diff
@@ -0,0 +1,124 @@
+(install-page)=
+
+# Installation
+
+(install-requirements)=
+
+## Requirements
+
+Nextflow can be used on any POSIX-compatible system (Linux, macOS, etc), and on Windows through [WSL](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux). It requires Bash 3.2 (or later) and [Java 11 (or later, up to 21)](http://www.oracle.com/technetwork/java/javase/downloads/index.html) to be installed. You can see which version you have using the following command:
+
+```bash
+java -version
+```
+
+We recommend that you install Java through [SDKMAN!](https://sdkman.io/), and that you use the latest LTS version of Temurin. See [this website](https://whichjdk.com/) for more information.
+
+To install Java with SDKMAN:
+
+1. Install SDKMAN:
+
+    ```bash
+    curl -s https://get.sdkman.io | bash
+    ```
+
+2. Open a new terminal and install Java:
+
+    ```bash
+    sdk install java 17.0.10-tem
+    ```
+
+3. Confirm that Java is installed correctly:
+
+    ```bash
+    java -version
+    ```
+
+4. To install Temurin 17:
+
+   ```bash
+   sdk install java 17.0.6-tem
+   ```
+
+(install-nextflow)=
+
+## Install Nextflow
+
+Nextflow is distributed as a self-installing package, in order to make the installation process as simple as possible:
+
+1. Install Nextflow:
+
+    ```bash
+    curl -s https://get.nextflow.io | bash
+    ```
+
+    This will create the `nextflow` executable in the current directory.
+
+    :::{tip}
+    You can set `export CAPSULE_LOG=none` to make the installation logs less verbose.
+    :::
+
+2. Make Nextflow executable:
+
+    ```bash
+    chmod +x nextflow
+    ```
+
+3. Move Nextflow into an executable path:
+
+    ```bash
+    sudo mv nextflow /usr/local/bin
+    ```
+
+4. Confirm that Nextflow is installed correctly:
+
+    ```bash
+    nextflow info
+    ```
+
+### Standalone distribution
```

Comment:
it would be better moving this after `Updates`, because is a *special* distribution, not supporting third-party plugins and updates